tests/functional/tests.py

```diff
@@ -52,7 +52,6 @@ def main():
     # client.trace_on(sys.stderr)
 
     # Make a new bucket.
-    bucket_name = 'minio-pytest'
```

Comment:
line 48:     bucket_name = uuid.uuid4().__str__() will generate a unique bucket name. Otherwise the make_bucket test will fail with hardcoded bucket name.